packages/currency/src/Balance.ts

```diff
@@ -35,10 +35,18 @@ type StringFormatOptions = {
   groupSeparator?: string
   showTicker?: boolean
   roundingMode?: BigNumber.RoundingMode
+  showTrailingZeroes?: boolean
 }
 
 const DC_TO_USD_MULTIPLIER = 0.00001
 
+const getVal = <T>(localValue?: T, defaultValue?: T) => {
+  if (localValue !== undefined) {
+    return localValue
+  }
+  return defaultValue
```

Comment:
```suggestion
  return localValue ?? defaultValue
```

Reply:
I was trying to remember this operator recently and couldn't remember what to google, much nicer than checking undefined

Reply:
@allenan @Perronef5 There was an issue with app center not liking it, so I put it out of my mind, but I believe it is fine now.